native/src/sagas/fetchResourceCache.ts

```diff
@@ -27,21 +22,21 @@ const createErrorMessage = (fetchResult: FetchResultType) => {
   )
 }
 
-function* watchOnProgress(): Generator<StrictEffect, void, number> {
+function* watchOnProgress() {
```

Comment:
Types for void are omitted?

Reply:
no. types for non-exported functions can be omitted.